MapboxNavigation.podspec

```diff
@@ -35,7 +35,7 @@ Pod::Spec.new do |s|
 
   # ――― Resources ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
 
-  s.resources = ['MapboxNavigation/Resources/**']
+  s.resources = ['MapboxNavigation/Resources/*/*']
```

Comment:
This new pattern excludes [Shields.plist](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/Resources/Shields.plist), which sits directly under Resources. Add another path to this array with just `*` to handle this case.